### ISSUE TITLE
Add a function for undoing all of the twitter blocks 

### ISSUE THREAD

**Issue by ethanmassie** [ethanmassie/twitterclean#3]
The blocked accounts don't need to remain blocked.  The initial block should remove them from your follower list.

**Comment by SyDan** [comment]
Just wanna confirm. Do you want to unblock all the handles that are blocked in a twitter account?

**Comment by ethanmassie** [comment]
Yes, there should be a command line option like `--undo-blocks` that will unblock blocked accounts.  I believe that if you block the user they won't be following your account anymore and unblocking them wont reapply their follow.
